pkgs/data/icons/nordzy-icon-theme/default.nix

```
@@ -0,0 +1,56 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+, gtk3
+, nordzy-theme-name ? "Nordzy"
+, nordzy-themes ? [ "all" ] # Override this to only install selected themes
+}:
+
+let
+  themes-arg-string = lib.strings.concatMapStrings (theme: "-t ${theme} ") nordzy-themes;
+in
+stdenvNoCC.mkDerivation {
+  pname = "nordzy-icon-theme";
+  version = "unstable-2021-12-14";
+
+  src = fetchFromGitHub {
+    owner = "alvatip";
+    repo = "Nordzy-icon";
+    rev = "5c247a4f19cf9849615631d1bf77727b945b634e";
+    sha256 = "Jqn5CF80xlYJ7H4qI1VEj91vcKPPoMXP5+sPs0ksiC4=";
+  };
+
+  # In the post patch phase we should first make sure to patch shebangs.
+  postPatch = ''
+    patchShebangs install.sh
+  '';
+
+  nativeBuildInputs = [
+    gtk3
+  ];
+
+  dontDropIconThemeCache = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/icons
+    ./install.sh --dest $out/share/icons \
```

```suggestion
    name= ./install.sh --dest $out/share/icons \
```
Unset the `name` environment variable to run the install script. This variable is used in the script and it should be a local variable to the script. nix sets it to the name of the package.